fuzzing/testdata/contracts/assertions/assert_allocate_too_much_memory.sol

```diff
@@ -0,0 +1,6 @@
+contract TestContract {
```

Comment:
For each solidity test file, can you add a comment at the top of it to highlight what the contract is doing? So for this one here is an example:

```
This contract ensures that the fuzzer will fail if too much memory is allocated
```